components/match2/commons/match_group_display_bracket.lua

```diff
@@ -24,7 +24,11 @@ local WikiSpecific = Lua.import('Module:Brkts/WikiSpecific', {requireDevIfEnable
 local OpponentDisplay = require('Module:OpponentLibraries').OpponentDisplay
 
 local html = mw.html
-local _NON_BREAKING_SPACE = '&nbsp;'
+local NON_BREAKING_SPACE = '&nbsp;'
+
+local DEFAULT_OPPONENT_HEIGHTS = {
+	duo = 2 * 17 + 6 + 4,
+}
```

Comment:
that would only work for duos
but if we add trio/quad it will fail because the 24 is incl. padding

would rather use soemthing like
```lua
local DEFAULT_OPPONENT_HEIGHT_NO_PADDING = 18
local OPPONENT_PADDING = 6
function BracketDisplay._computeOpponentHeight(opponentType)
	return (Opponent.partySize(opponentType) or 1) * DEFAULT_OPPONENT_HEIGHT_NO_PADDING + OPPONENT_PADDING
end
```

Reply:
Well `(2 * props.config.opponentHeight) - (4 * (NumOpponent - 1)`, but since it was specific for duos.
Alternatively adjust what height means (and updated the usages). But it should still be fetched from config, not hardcoded.

Reply:
prefer to write it as `NumOpponent * (props.config.opponentHeight - 4) + 4` (making the 4 a const var) as it is clearer what happens imo (taking away the padding from the opponent multiplying it with the number of rows/players and after that adding the padding once)